plugins/star-rating/api/api.js

```diff
@@ -177,12 +178,12 @@ var plugin = {},
                                 "email": currEvent.segmentation.email,
                                 "comment": currEvent.segmentation.comment,
                                 "ts":currEvent.timestamp,
-                                "device_id":params.qstring.device_id,
+                                "device_id":currEvent.device_id,
```

Comment:
The way I see in popup dialog is that device_id is sent through `params.qstring.device_id` and not currEvent